drift/views/v0.py

```diff
@@ -25,8 +25,11 @@ def compare():
     if auth_key is None:
         raise HTTPError(HTTPStatus.BAD_REQUEST, message="auth header not specified")
 
-    comparisons = info_parser.build_comparisons(fetch_hosts(host_ids, auth_key))
-    return jsonify(comparisons)
+    try:
+        comparisons = info_parser.build_comparisons(fetch_hosts(host_ids, auth_key))
+        return jsonify(comparisons)
+    except SystemNotReturned as error:
+        raise HTTPError(HTTPStatus.BAD_REQUEST, message=error.message)
```

Comment:
Does this return a string or JSON? If a string, do we want to standardize on a JSON formatted error message format? Are there any standards within the platform on error returns?

Reply:
it returns a json:

```
{"message":"System d1d22ce7-4e52-4767-867a-2ccac52485ec not available to display"}
```

Reply:
@ehelms the platform docs have some differences with the way the API is right now. I'll put in a new card for it, but it's something that is going to be bigger than this PR.

Reply:
And last point of clarity, this will 'fully error out' as in, instead of just showing any systems we can, right now it will bomb out? I think that's fine. Just making sure I follow your idea here. We should probably talk to Rox about this interaction next sprint.

Reply:
correct, if any UUID is not found it will bail on the whole thing. It will also error on the first UUID and not proceed with fetching the rest.